### ISSUE TITLE
"Cert not yet due for renewal", but it expires on 16th January 2017 (It's 27th Dec. today)

### ISSUE THREAD

**Issue by adnanberberovic** [certbot/certbot#3962]
Basically "certbot renew" should renew all certificates which expire within 30 days. My certificate does (my site is www.adnanber.be so you can check the certificate there), but certbot insists that it is not due for renewal (I even got an e-mail notification saying that it expires in 19 days).

The output when running certbot renew is:
```
Saving debug log to /var/log/letsencrypt/letsencrypt.log

-------------------------------------------------------------------------------
Processing /etc/letsencrypt/renewal/adnanber.be.conf
-------------------------------------------------------------------------------
Cert not yet due for renewal

The following certs are not due for renewal yet:
  /etc/letsencrypt/live/adnanber.be/fullchain.pem (skipped)
No renewals were attempted.
```

A little block from my nginx file (/etc/nginx/sites/adnanber.be)
```
    #ssl
    ssl on;
    ssl_certificate /etc/letsencrypt/live/adnanber.be/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/adnanber.be/privkey.pem;
```

My renewal file (/etc/letsencrypt/renewal/adnanber.be.conf)

```
# renew_before_expiry = 30 days
cert = /etc/letsencrypt/live/adnanber.be/cert.pem
privkey = /etc/letsencrypt/live/adnanber.be/privkey.pem
chain = /etc/letsencrypt/live/adnanber.be/chain.pem
fullchain = /etc/letsencrypt/live/adnanber.be/fullchain.pem
version = 0.9.3

# Options and defaults used in the renewal process
[renewalparams]
installer = None
authenticator = webroot
webroot_path = /etc/letsencrypt/live,
account = [REDACTED]
[[webroot_map]]
www.adnanber.be = /etc/letsencrypt/live
adnanber.be = /etc/letsencrypt/live
```

I'm not sure if the first row is supposed to be commented, or if there's supposed to be a comma at the end of the webroot_path row, but I tried removing them and got the same error, so I put them back like they used to be.

Anyway, certbot has worked fine for me for the past 2 or 3 times I used it to create and then renew. For some reason it just suddenly doesn't work now. I might just have done something retarded but I do need some help and guidance in how to solve this issue.

I've also restarted my nginx server and my remote machine, didn't help.

EDIT: Ok on my page it says 27th January, but the e-mail notification I got says 16th January. I'll wait until tomorrow and then run renew again.

**Comment by bmw** [comment]
Any update?

**Comment by adnanberberovic** [comment]
Yes, sorry about that! 
Well I received the wrong e-mail because I still had my old certificate renew config file, from when I only ran certbot with -d adnanber.be, and the last time I renewed I had both configuration files (one for www.adnanber.be and adnanber.be, and one for only adnanber.be). I've removed the old one, and from now on I should only get e-mail notifications regarding my one certificate for both www.adnanber.be and adnanber.be.
